### DISCUSSION TITLE
How to add an alert or header informational message to the dashboard?

### DISCUSSION THREAD

**Discussion by hkdatalore** [gogs/gogs#6883] [Q&A]
I've looked through the documentation and the source templates and I cannot figure out how to do this.

I would like to add an informational message to show on the dashboard indicated by the red line.  How would I accomplish that?

![image](https://user-images.githubusercontent.com/16761638/161087407-888dbeb3-4f8f-4802-966e-63b81c33352f.png)

**Comment by unknwon** [comment]
As of now, you'd need to use [custom templates](https://gogs.io/docs/features/custom_template#override-html-templates) feature to archive this, and add something between these two lines: 

https://github.com/gogs/gogs/blob/3f439f15c3a54e144009233e277f7eee49eb1515/templates/user/dashboard/dashboard.tmpl#L6-L7

A little bit dirty hack, frankly.

Meanwhile, you can also make a feature request for the global banner.
